### PR TITLE
Update gwemopt

### PR DESCRIPTION
**Pull request by mcoughlin**
**Does this pull request make any changes to the database?**
No

Code changes that affect the database require special attention for data migration.
If your change affects the database, describe the migration plan here.

### CODE REVIEW

File: growth/too/tasks/tiles.py [hunk 1]
```diff
@@ -198,6 +198,7 @@ def params_struct(dateobs, tobs=None, filt=['r'], exposuretimes=[60.0],
     params["mindiff"] = mindiff
 
     params = gwemopt.segments.get_telescope_segments(params)
+    params = gwemopt.utils.params_checker(params)
```
Comment (lpsinger) [inline growth/too/tasks/tiles.py]: What does `params_checker` do?

File: growth/too/tasks/tiles.py [hunk 2]
```diff
@@ -240,9 +241,10 @@ def gen_structs(params):
         raise ValueError(
             'Need tilesType to be moc, greedy, hierarchical, galaxy or ranked')
 
-    coverage_struct = gwemopt.coverage.timeallocation(params,
-                                                      map_struct,
-                                                      tile_structs)
+    tile_structs, coverage_struct =\
```
Comment (lpsinger) [inline growth/too/tasks/tiles.py]: Please add missing space after binary operand.

File: growth/too/tasks/tiles.py (hunk 1 above)
Comment (mcoughlin) [reply growth/too/tasks/tiles.py]: It chooses sensible defaults for parameters that are not uniquely specified.

File: growth/too/tasks/tiles.py (hunk 2 above)
Comment (mcoughlin) [reply growth/too/tasks/tiles.py]: flake8 --show-source growth passes for me?

Comment (lpsinger) [reply growth/too/tasks/tiles.py]: Nonetheless, please add a space.

File: growth/too/tasks/tiles.py (hunk 1 above)
Comment (lpsinger) [reply growth/too/tasks/tiles.py]: Why is this a new function? Why don't the existing functions provide sensible defaults?

Comment (mcoughlin) [reply growth/too/tasks/tiles.py]: In general, they do, with a few exceptions that arise from the command line version of the tool setting them. This helps rectify this.

File: growth/too/tasks/tiles.py (hunk 2 above)
Comment (mcoughlin) [reply growth/too/tasks/tiles.py]: I cannot make flake8 pass with this.

**Pull request closed without merging**